deno/lib/README.md

```diff
@@ -623,7 +623,7 @@ z.coerce.boolean().parse(null); // => false
 
 ## Literals
 
-Literal schemas represent a [literal type](https://www.typescriptlang.org/docs/handbook/literal-types.html), like `"hello world"` or `5`.
+Literal schemas represent a [literal type](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-types), like `"hello world"` or `5`.
```

Comment:
Not sure why this is showing up, seems like it's [already in master](https://github.com/colinhacks/zod/commit/51e14beeab2f469fcbf18e3df44653e1643f5487)?